contracts/SpartanCouncil.sol

```diff
@@ -12,8 +12,8 @@ contract SpartanCouncil is Ownable {
     event Mint(uint256 indexed tokenId, address to);
     // Event that is emitted when an existing SpartanCouncil token is burned
     event Burn(uint256 indexed tokenId);
-    // Event that is emitted when an existing SpartanCouncil token is transferred
-    event Transferred(address indexed from, address indexed to, uint256 indexed tokenId);
+    // Event that is emitted when an existing SpartanCouncil token is Transferred
+    event Transfer(address indexed from, address indexed to, uint256 indexed tokenId);
```

Comment:
@andytcf  

Following ERC20 standard, 

`event Transfer(address indexed _from, address indexed _to, uint256 _value)` the value might be hardcoded to 1 here instead of the `tokenId` if etherscan is parsing this for the holder balance.

Reply:
https://docs.openzeppelin.com/contracts/2.x/api/token/erc721#IERC721-Transfer-address-address-uint256-

Reply:
```Transfer(address from, address to, uint256 tokenId)```

Reply:
https://ethereum.stackexchange.com/questions/59862/how-etherscan-io-is-able-to-track-erc721-tokens-on-the-blockchain